builder/images/base-win64/Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM $GH_REPO/base:latest
 
 RUN --mount=src=ct-ng-config,dst=/.config \
     git clone --filter=blob:none https://github.com/crosstool-ng/crosstool-ng.git /ct-ng && cd /ct-ng && \
+    curl "https://gcc.gnu.org/git/?p=gcc.git;a=patch;h=747c4b58573ea00419f64293a61537eb69f43307" > packages/gcc/14.1.0/9999-fix-inline.patch && \
```

Comment:
Let's hope gnu.org won't down again due to this. Can we just self-contain this patch?

Reply:
let's use the gh mirror https://github.com/gcc-mirror/gcc/commit/747c4b58573ea00419f64293a61537eb69f43307